### DISCUSSION TITLE
Failed to retrieve tables Error

### DISCUSSION THREAD

**Discussion by baekspace88** [supabase/supabase#21007] [Questions]
My team has been unable to access the Supabase database because we failed to retrieve tables. The error message we received states: 'Connection terminated due to connection timeout.' This issue has persisted since this morning. Given that it is a real database server, resolving this is urgent. Please inform me of both the reason for this problem and the solution.

**Comment by baekspace88** [comment]
I checked the issue had been solved just now

**Reply by GaryAustin1** [reply]
In the future for issues such as this contacting support is the main avenue
Checking status before hand is also useful.  Here is likely your issue:  https://status.supabase.com/incidents/1bl9pwv07k6g  But at the start of an event it can take awhile (especially if low volume of reports).

Github and Discord are not direct contacts to Supabase and there are no guarantees issues will be noticed be noticed by someone who can help.
